File: sms-api-java/auth-service/src/main/java/com/ffos/authservice/controller/dto/UserDto.java
package com.ffos.authservice.controller.dto;

import lombok.Builder;
import lombok.Data;
import lombok.Getter;
import lombok.Setter;
import org.springframework.stereotype.Component;

@Component
@Data
@Getter
@Setter
public class UserDto {
    private long idUser;
    private String username;
    private String password;
    public UserDto() {
    }

}

File: sms-api-kotlin/sendmessageservice/src/main/kotlin/com/example/sendmessageservice/controller/dto/UserDto.kt
package com.example.sendmessageservice.controller.dto

data class UserDto(
    var idUser : Long? = null,
    var firstName: String? = null,
    var lastName: String? = null,
    var username: String? = null,
    var mail: String? = null,
    var password: String? = null,
    var number: String? = null
)